app/presenters/publishing_api_presenters/take_part.rb

```diff
@@ -0,0 +1,52 @@
+module PublishingApiPresenters
+  class TakePart
+    attr_reader :take_part_page, :update_type
+
+    def initialize(take_part_page, options = {})
+      @take_part_page = take_part_page
+      @update_type = options[:update_type] || 'major'
```

Comment:
I think it needs to keep the boolean as sometimes the key is present but the value is nil in which case I think we need to default to 'major'.